### PR TITLE
Replace github paths in `html_inline` links/images

### LINKED ISSUE #304: need inline-html rule for the `github` plugin
currently, `github` doesn't work for img/links written in inline html

### PR DESCRIPTION
**Pull request by revin**
Fixes #304

### CODE REVIEW

File: lib/plugin/github.js [hunk 1]
```diff
@@ -54,4 +82,23 @@ module.exports = function (md, opts) {
...
     }
   }
+
+  // rewrite image locations in inline HTML to be fully qualified github URLs
+  var originalHtmlInlineRule = md.renderer.rules.html_inline
+  md.renderer.rules.html_inline = function (tokens, idx, options, env, self) {
+    var content = tokens[idx].content
+    var repoPath = path.join(repo.user, repo.repo, 'master')
+    var imageBaseUrl = 'https://raw.githubusercontent.com/' + repoPath
+    var newHtml
```
Comment (revin) [inline lib/plugin/github.js]: I'm not super happy with the nested `if()` structure here, but just doing, e.g.,
```js
newHtml = replaceTagAttribute(content, 'img', 'src', imageBaseUrl + '/') ||
          replaceTagAttribute(content, 'a', 'href', repo.https_url + '/blob/master/')
```
looked bad when the `standard` formatting was applied (leading whitespace on the second line gets removed)

**Comment by ashleygwilliams** [comment]
assuming that's the case (which if it weren't would mean that we had bigger problems to begin with)- this looks great for merge 🐬 ✨

**Comment by revin** [comment]
Yes it's only for GH repos thanks to `github-url-to-object` doing the URL parsing (see [github.js#L50-52](https://github.com/npm/marky-markdown/blob/2dbdca0579038b50bc7ae4764beea36f5e972a8f/lib/plugin/github.js#L50-L52)); it returns `null` for non-GH URLs

**Comment by ashleygwilliams** [comment]
a+. mergin!

**Pull request merged**